src/app/admin/components/file-deletion/file-hide-or-delete/file-hide-or-delete.component.ts

```diff
@@ -53,14 +55,23 @@ export class FileHideOrDeleteComponent implements OnInit {
     audioFile: AssociatedMedia[];
   } = { media: [], audioFile: [] };
 
-  audioHideComplete = false;
-
   id = +this.route.snapshot.params.id;
 
-  isSubmitted = false;
-  isAssociatedAudio = false;
+  audioHideComplete = signal(false);
+  isSubmitted = signal(false);
+  isAssociatedAudio = signal(false);
   continueLink = this.getContinueLink();
 
+  eff = effect(() => {
+    if (this.isAssociatedAudio() && !this.audioHideComplete()) {
+      this.title.setTitle('DARTS Associated Audio Files');
+    } else if (this.audioHideComplete()) {
+      this.title.setTitle('DARTS Check For Associated Files');
```

Comment:
@davet1985 do we want to show same title for hiding/deleting transcripts too as both flows show same screen?

Reply:
tis done

Reply:
I'm not convinced this list of dynamic titles is exhaustive either